ibm/service/power/resource_ibm_pi_instance_console_language.go

```diff
@@ -32,12 +30,12 @@ func ResourceIBMPIInstanceConsoleLanguage() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			// Required Attributes
-			helpers.PICloudInstanceId: {
+			Arg_CloudInstanceID: {
 				Type:        schema.TypeString,
 				Required:    true,
 				Description: "PI cloud instance ID",
 			},
-			helpers.PIInstanceName: {
+			Arg_InstanceName: {
 				Type:        schema.TypeString,
 				Required:    true,
 				Description: "The unique identifier or name of the instance",
```

Comment:
Create a new constant for PIConsoleLanguageCode and put into the constant file.

Reply:
Added